### DISCUSSION TITLE
milvus集群部署

### DISCUSSION THREAD

**Discussion by jhyever** [milvus-io/milvus#38482] [Announcement]
1、![image](https://github.com/user-attachments/assets/fe388aab-5c46-41d9-b1ae-e6493186d6fc)、
2、这个有问题
![image](https://github.com/user-attachments/assets/11ae8a52-1ed5-428c-b1f8-91aeca0f1231)
但是这个又正常running
![image](https://github.com/user-attachments/assets/7773cbcb-97c4-4c6b-8db0-45983207aaa8)
3、
![image](https://github.com/user-attachments/assets/3c1348ee-a9b0-4eb2-9b47-b40a7d9852f8)
麻烦帮忙看看

**Comment by LoveEachDay** [comment]
@jhyever You'd better first ensure that Milvus dependency pods, such as etcd, MinIO, and Pulsar, are running smoothly.
